source/aggregation-pipeline-builder.txt

```diff
@@ -44,173 +44,229 @@ Create an Aggregation Pipeline
 
 .. include:: /includes/steps/create-agg-pipeline.rst
 
+Specify Custom Collation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use :manual:`custom collation </reference/collation/>` to specify
+language-specific rules for string comparison, such as rules for
+letter-case and accent marks.
+
+To specify a custom collation:
+
+1. Click the :guilabel:`Collation` button at the top of the pipeline
+   builder.
+
+#. Enter your collation document.
+
+Limitations
+~~~~~~~~~~~
+
+The :pipeline:`$out` stage is not available if you are connected to a
+`Data Lake <https://www.mongodb.com/atlas/data-lake>`__. 
+
 .. _save-agg-pipeline:
 
 Save a Pipeline
 ---------------
 
-To save a pipeline:
+You can save your pipeline so it can be reloaded into
+the pipeline builder in the future. You can also create a
+:manual:`view </core/views/>` from the results of your pipeline.
 
-1. Name your pipeline using the
-   :guilabel:`Enter a pipeline name` input at the top of the
-   :guilabel:`Aggregations` view.
+To save your pipeline:
 
-2. Click :guilabel:`Save Pipeline`.
+1. Click the :guilabel:`Save` button at the top of the pipeline
+   builder.
 
-.. _open-saved-pipeline:
+#. Enter a name for your pipeline.
 
-Open a Saved Pipeline
----------------------
+#. Click :guilabel:`Save`.
 
-.. important::
+Create a View from Pipeline Results 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   Opening a saved pipeline abandons any unsaved changes to the
-   pipeline you are currently working on. If the circle at the
-   top-right of the :guilabel:`Aggregations` view is orange, this
-   means you have unsaved changes to your pipeline.
+.. note::
 
-To open a previously saved pipeline:
+   Creating a view from pipeline results does not save the pipeline
+   itself.
 
-1. Click the :guilabel:`Folder` icon to the left of the pipeline name
-   input.
+To create a view from your pipeline results:
 
-2. Hover over the pipeline you want to open and click :guilabel:`Open`.
+1. Click the arrow next to the :guilabel:`Save` button at the top
+   of the pipeline builder.
 
-3. In the ensuing dialog, click :guilabel:`Open Pipeline`.
+#. Click :guilabel:`Create a View`.
+
+#. Enter a name for your view.
+
+#. Click :guilabel:`Create`.
+
+|compass-short| creates a view from your pipeline results in the same
+database where the pipeline was created.
+
+.. _open-saved-pipeline:
 
-Additional Pipeline Options
----------------------------
+Open a Saved Pipeline
+---------------------
 
-Click the :guilabel:`...` button to the right of the
-:guilabel:`Save Pipeline` button to access additional pipeline
-options:
+1. Click the :guilabel:`Folder` icon at the top left of the
+   pipeline builder.
 
-.. figure:: /images/compass/agg-builder-export-dropdown.png
-   :alt: Aggregation Builder additional pipeline options
+2. Hover over the pipeline you want to open and click :guilabel:`Open`.
 
-The following additional options are available:
+3. In the ensuing dialog, click :guilabel:`Open Pipeline`.
```

Comment:
'ensuing' sounds jargony to me.  Consider deleting it.  Also, we tend to use 'modal' rather than 'dialog'.